splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SQLWarningIT.java

```diff
@@ -0,0 +1,59 @@
+package com.splicemachine.derby.impl.sql.execute.operations;
+
+import com.splicemachine.derby.test.framework.SpliceSchemaWatcher;
+import com.splicemachine.derby.test.framework.SpliceUnitTest;
+import com.splicemachine.derby.test.framework.SpliceWatcher;
+import com.splicemachine.test_tools.TableCreator;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLWarning;
+import java.sql.Statement;
+
+import static com.splicemachine.test_tools.Rows.row;
+import static com.splicemachine.test_tools.Rows.rows;
+
+public class SQLWarningIT extends SpliceUnitTest {
+    public static final String CLASS_NAME = SQLWarningIT.class.getSimpleName().toUpperCase();
+
+    protected static SpliceWatcher spliceClassWatcher = new SpliceWatcher(CLASS_NAME);
+    protected static SpliceSchemaWatcher spliceSchemaWatcher = new SpliceSchemaWatcher(CLASS_NAME);
+
+    @ClassRule
+    public static TestRule chain = RuleChain.outerRule(spliceClassWatcher)
+            .around(spliceSchemaWatcher);
+    @BeforeClass
+    public static void setup() throws Exception {
+        setup(spliceClassWatcher);
+    }
+
+    private static void setup(SpliceWatcher spliceClassWatcher) throws Exception {
+        Connection conn = spliceClassWatcher.getOrCreateConnection();
+        new TableCreator(conn)
+                .withCreate("create table A (c1 int)")
+                .withInsert("insert into A values(?)")
+                .withRows(rows(
+                        row(1),
+                        row(2))).create();
+    }
+
+    @Test
+    public void TestNoRowsAffectedWarning() throws Exception {
```

Comment:
can you add a negative test case with `splice.db2.error.compatible` set to false?

Reply:
method name should start with lowercase